src/lumigo_tracer/lumigo_utils.py

```diff
@@ -156,12 +156,17 @@ def config(
     :param edge_kinesis_aws_access_key_id: The credentials to push to the Kinesis in China region
     :param edge_kinesis_aws_secret_access_key: The credentials to push to the Kinesis in China region
     """
+    Configuration.token = token or os.environ.get(LUMIGO_TOKEN_KEY, "")
+    if not (Configuration.token and re.match("[t][_][[a-z0-9]{21}", Configuration.token)):
+        warn_client(
+            "Invalid token used, copy your token from Settings → Tracing from Lumigo’s platform"
+        )
```

Comment:
can you please also stop the tracer (set `should_report=False`), so we will not send corrupted data to lumigo?